charts/prometheus-thanos/values.yaml

```diff
@@ -67,6 +67,24 @@ querier:
     successThreshold: 1
     timeoutSeconds: 30
 
+  autoscaling:
+    enabled: true
```

Comment:
I would prefer to disable the horizontal scaling by default, otherwise users might facing issues if they have setup vertical pod autoscaler in their cluster.

Reply:
Thanks, excellent catch. Autoscaling was meant to be disabled and the change was accidentally committed.